os-eap-probes/added/readinessProbe.sh

```diff
@@ -8,7 +8,7 @@ LOG=/tmp/readiness-log
 
 COUNT=30
 SLEEP=5
-DEBUG=false
+DEBUG=${SCRIPT_DEBUG:-false}
```

Comment:
This seems out of scope, but innocuous.

Reply:
I think this helps quite nice during investigation. The `cct_module` defines the `SCRIPT_DEBUG` env variable to cause `set -x` to be set up. Here we got debug information from the python readiness probe as well when the env variable is set. I used that with success while investigating errors.